roles/docker/templates/daemon.json

```diff
@@ -60,7 +60,7 @@
     "tlskey": "",
     "tlsverify": true,
     "userns-remap": "",
-    "add-registry": {{  docker_additional_registries|default([]) }},
-    "blocked-registries": {{ docker_blocked_registries|default([]) }},
+    "add-registry": {{ l_docker_additional_registries | default([]) }},
+    "blocked-registries": {{ l_docker_blocked_registries|default([]) }},
     "userland-proxy-path": "/usr/libexec/docker/docker-proxy-current"
 }
```

Comment:
This might only fixes `unicode` issue. We definitely would hit other issues (please see `Additional info` I pasted in that bug description) even if this one is fixed.